mitosheet/mitosheet/is_type_utils.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any, List, Tuple, Union
+import pandas as pd
+
+"""
```

Comment:
Since these are helper functions used in our formulas, to be safe do we want to put them in the v1 folder? Their results are unlikely to change, in fairness. 

Would love to hear your thoughts though on what we want to version and what we don't want to version.

Reply:
We generally want to version _everythign that might change_. These... I don't think will change. They are also used everywhere in our codebase, so versioning them is a real pain. So I think we leave them as is.